src/com/company/Board.java

```diff
@@ -28,12 +42,29 @@ public boolean insertSign(final PlayerSignType sign, final int row, final int co
         return false;
     }
 
-    public String getSignText(final int row, final int column) {
-        if (data[row][column] == PlayerSignType.EMPTY) {
-            return (" ");
-        } else {
-            return data[row][column].toString();
+    public void addSignValue(final int row, final int column) {
+        rowValue[row] += data[row][column].intValue;
+        columnValue[column] += data[row][column].intValue;
+        if (row == column) {
+            diagXxValue += data[row][column].intValue;
+        }
+        if (column == data.length - row - 1) {
+            diagYyValue += data[row][column].intValue;
+        }
+    }
+
+    public boolean checkWinner(final int row, final int column) {
+        if (abs(rowValue[row]) == size
```

Comment:
Znów mamy ten przypadek w którym "jeśli to prawda to zwróć prawdę, jeśli to fałsz to zwróć fałsz"... popraw proszę.

Reply:
Poprawione.